source/fuzz/fuzzer_pass_add_equation_instructions.cpp

```diff
@@ -64,12 +64,76 @@ void FuzzerPassAddEquationInstructions::Apply() {
 
         // Try the opcodes for which we know how to make ids at random until
         // something works.
-        std::vector<SpvOp> candidate_opcodes = {SpvOpIAdd, SpvOpISub,
-                                                SpvOpLogicalNot, SpvOpSNegate};
+        std::vector<SpvOp> candidate_opcodes = {
+            SpvOpIAdd,        SpvOpISub,        SpvOpLogicalNot,
+            SpvOpSNegate,     SpvOpConvertUToF, SpvOpConvertSToF,
+            SpvOpConvertFToU, SpvOpConvertFToS};
         do {
           auto opcode =
               GetFuzzerContext()->RemoveAtRandomIndex(&candidate_opcodes);
           switch (opcode) {
+            case SpvOpConvertFToS:
+            case SpvOpConvertFToU:
+            case SpvOpConvertSToF:
+            case SpvOpConvertUToF: {
+              auto candidate_instructions =
+                  opcode == SpvOpConvertFToS || opcode == SpvOpConvertFToU
```

Comment:
I would be inclined to put "SpvOpConvertFToS || opcode == SpvOpConvertFToU" in parentheses for readibility.

Reply:
Fixed in e4b2f8c.